### PR TITLE
Feature/volunteer edit data

### PR DESCRIPTION
**Pull request by ghost**
I've also updated the welcome email.

### CODE REVIEW

File: app/views/devise/registrations/edit.html.slim [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        = "(#{@minimum_password_length} characters minimum)"    
+
+  div
+    = f.label :password_confirmation 
+    br
+    = f.password_field :password_confirmation, autocomplete: "off" 
+
+  div 
+    = f.label :current_password  
+    i
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: remove extra `i`. Think this doesn't do anything.

File: app/views/devise/registrations/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    br
+    = f.collection_check_boxes :target_group_ids, TargetGroup.all, :id, :name
+
+  div
+    = f.label :'Skills you can offer'
+    br
+    = f.collection_check_boxes :skill_ids, Skill.all, :id, :name
+
+  div
+    =f.label :other_talents
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: add space between `=` and `f` for consistency

File: app/views/devise/registrations/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+  div 
+    = f.label :email 
+    br
+    = f.email_field :email, autofocus: true 
+  
+
+  - if devise_mapping.confirmable? && resource.pending_reconfirmation? 
+    div
+      Currently waiting confirmation for: - resource.unconfirmed_email
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: think you need a `|` in front

File: app/views/devise/registrations/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    = f.text_area :about_me
+
+  div
+    = f.label :password 
+    p 
+      |(leave blank if you don't want to change it)
+    = f.password_field :password, autocomplete: "off" 
+    - if @minimum_password_length 
+    em
+        = "(#{@minimum_password_length} characters minimum)"
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: think you need to indent the above line and this? you can also do `| (#{minimum_password_length)} characters minimum)`

File: app/views/devise/registrations/edit.html.slim [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    = f.label :current_password  
+    i
+    p 
+      | (we need your current password to confirm your changes)
+    = f.password_field :current_password, autocomplete: "off" 
+  
+  .actions
+    = f.submit "Update" 
+  
+
```
Comment (tbssc) [inline app/views/devise/registrations/edit.html.slim]: remove two extra lines

File: app/views/devise/mailer/confirmation_instructions.html.erb [hunk 6]
```diff
@@ -2,4 +2,7 @@
```
Comment (tbssc) [inline app/views/devise/mailer/confirmation_instructions.html.erb]: we can also convert this to `slim` 😉

**Review by tbssc — CHANGES_REQUESTED** [review]
this looks good! just a few changes.

File: app/views/devise/mailer/confirmation_instructions.html.slim [hunk 7]
```diff
@@ -0,0 +1,9 @@
+  p Welcome = @email
+
+  p You can confirm your account email through the link below:
+
+  = link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token)
+
+  p 
+    |Thank you for your interest to be a PPIS Volunteer! We'll contact you if there's any suitable volunteer opportunities.
```
Comment (anthonysherwinchua) [inline app/views/devise/mailer/confirmation_instructions.html.slim]: space between the `|` and the first letter please

**Review by anthonysherwinchua — APPROVED** [review]
LGTM! 🚀

**Review by anthonysherwinchua — APPROVED** [review]

**Pull request merged**